docs/fips.md

```diff
@@ -0,0 +1,195 @@
+# BoringCrypto and FIPS 140-3 compliance
```

Comment:
Where did you get FIPS 140-3 from? The last module validation of Boring I see is still against FIPS 140-2? (https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4407)

Maybe just say "FIPS compliance" and omit the 140-3?

Reply:
I swear I saw a cert for 140-3, but it's clearly not actually there. Sorry for the confusion.